pegasus/sites.v3/code.org/public/educate/curriculum/teacher-led.md

```diff
@@ -139,10 +139,7 @@ Your class will be be creating a 'history of computers' web page/Scratch project
 **Ages 12-16. Science (Climate), Science (Environment), Unplugged.** Students draw a picture, and take turns giving the class steps to recreate their drawing.
  
 [Mozilla Homework Excuse Generator](https://d157rqmxrxj6ey.cloudfront.net/mozillalearning/11701/)  
-**Ages 13-18. Programming, Language Arts, Web-based.** Use Mozilla's code editor, Thimble, to edit strings inside JavaScript arrays and customize the homework excuse generators. 
-
-[Fact or Fiction?](http://www.vizwik.com/hoc)  
-**Ages 16-18. Programming, Build an App, Web-based.** Students create an app to survey whether their classmates think a statement is find a fact or fiction.
+**Ages 13-18. Programming, Language Arts, Web-based.** Use Mozilla's code editor, Thimble, to edit strings inside JavaScript arrays and customize the homework excuse generators.
```

Comment:
I don't know why GitHub isn't recognizing it but I just removed a trailing space.